pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools","setuptools_scm","swig"]
+requires = ["setuptools","setuptools_scm","swig==4.2.1.post0"]

```

Comment:
Why would we fix this to only 1 specific version of swig? <4.3.0 would work. But we really should just sync with upstream, so I will do that.

Reply:
Ok

Reply:
updated.